backend/api/v1/views.py

```diff
@@ -1,34 +1,59 @@
 import json
 import os
+import shutil
+from glob import glob
 from urllib.parse import urljoin
-
-from flask import Blueprint, jsonify, abort, send_file, url_for, current_app, redirect, request
+from datetime import datetime, timezone
+
+from flask import (
+    Blueprint,
+    jsonify,
+    abort,
+    send_file,
+    url_for,
+    current_app,
+    redirect,
+    request,
+)
 from werkzeug import exceptions
 
-from utils.api_utils import authenticated
-from utils import grader_utils
-from .models import Content
+from utils.api_utils import (
+    authenticated,
+    get_user_assignment_folder,
+    get_storage_folder,
+    slugify,
+    is_grader,
+)
+from .models import Content, Submission, Notebook
+from app import db
 
 
 # Create a blueprint
-blueprint = Blueprint('api_v1', __name__)
+blueprint = Blueprint("api_v1", __name__)
 
 # User route: show the user information
-@blueprint.route('/user')
+@blueprint.route("/user", methods=["GET"])
 @authenticated
 def show_user(user):
+    """
+    Get information on the currently authenticated user.
+    """
     return_dict = {
         "isAdmin": user["admin"],
         "created": user["created"],
         "lastActivity": user["last_activity"],
         "name": user["name"],
     }
-    return jsonify(user)
+    return jsonify(return_dict)
```

Comment:
In case you want to have this shorter you can also directly return the dict: https://flask.palletsprojects.com/en/1.1.x/quickstart/#apis-with-json